telegram/utils/helpers.py

```diff
@@ -113,3 +120,45 @@ def mention_markdown(user_id, name):
     """
     if isinstance(user_id, int):
         return '[{}](tg://user?id={})'.format(escape_markdown(name), user_id)
+
+
+def _extract_urls_from_text(text):
```

Comment:
Please make this function a generator (i.e. use `yield` instead of returning a list)

Reply:
actually, this is a dead code. maybe just remove this function?